clients/gitlabrepo/search_test.go

```diff
@@ -125,3 +129,69 @@ func TestBuildQuery(t *testing.T) {
 		})
 	}
 }
+
+func Test_search(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		name         string
+		responsePath string
+		want         clients.SearchResponse
+		wantErr      bool
+	}{
+		{
+			name:         "valid search",
+			responsePath: "./testdata/valid-search-result",
+			want: clients.SearchResponse{
+				Results: []clients.SearchResult{
+					{
+						Path: "README.md",
+					},
+				},
+				Hits: 1,
+			},
+
+			wantErr: false,
+		},
```

Comment:
can you add a test which returns 0 results?

Reply:
👍